components/Members/MembersTabs.tsx

```diff
@@ -17,12 +17,14 @@ interface MembersTabsProps {
   activeTab: Member
   onChange: (x) => void
   tabs: Array<Member>
+  isNftMode?: boolean
 }
 
 const MembersTabs: FunctionComponent<MembersTabsProps> = ({
   activeTab,
   onChange,
   tabs,
+  isNftMode,
```

Comment:
this doesn't really need to be a prop because this component can just use hooks and figure this out on its own, but whatever